<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProgramDialog</class>
 <widget class="QDialog" name="AddProgramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>أضف برنامج الى قائمة الحظر</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>323</height>
       </rect>
      </property>
      <attribute name="label">
       <string>أضف برنامج من خلال قائمة البرامج الموجودة على النظام</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>323</height>
       </rect>
      </property>
      <attribute name="label">
       <string>أضف برنامج من خلال تحديد مساره على النظام أو كتابة اسمه</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>اسم البرنامج</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>استعراض</string>
           </property>
           <property name="icon">
            <iconset resource="CoEIAHajeb.qrc">
             <normaloff>:/images/category-empty.png</normaloff>:/images/category-empty.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>318</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>إلغاء</string>
       </property>
       <property name="icon">
        <iconset resource="CoEIAHajeb.qrc">
         <normaloff>:/images/disable.png</normaloff>:/images/disable.png</iconset>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>إضافة</string>
       </property>
       <property name="icon">
        <iconset resource="CoEIAHajeb.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="CoEIAHajeb.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddProgramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
